packages/ui/src/containers/PasscodeValidation/index.tsx

```diff
@@ -61,9 +61,16 @@ const PasscodeValidation = ({ type, method, className, target }: Props) => {
     verifyPasscodeErrorHandlers
   );
 
-  const { result: sendPasscodeResult, run: sendPassCode } = useApi(
-    getSendPasscodeApi(type, method)
-  );
+  const { run: sendPassCode } = useApi(getSendPasscodeApi(type, method));
+
+  const resendPasscodeHandler = useCallback(async () => {
```

Comment:
Can we even remove the `useCallback`?

Reply:
hmm not sure.   since I have a count down on this page, without useCallback would it re-define the handler method in every render cycle?